lib/Runtime/Library/GlobalObject.cpp

```diff
@@ -1748,11 +1748,11 @@ namespace Js
         return FALSE;
     }
 
-    BOOL GlobalObject::HasProperty(PropertyId propertyId)
+    PropertyQueryFlags GlobalObject::HasPropertyQuery(PropertyId propertyId)
     {
-        return DynamicObject::HasProperty(propertyId) ||
+        return JavascriptConversion::BooleanToPropertyQueryFlags(JavascriptConversion::PropertyQueryFlagsToBoolean(DynamicObject::HasPropertyQuery(propertyId)) ||
```

Comment:
Yes - see other comment.

---
In reply to: [106305579](https://github.com/Microsoft/ChakraCore/pull/2640#discussion_r106305579) [](ancestors = 106305579)